### ISSUE TITLE
error while loading shared libraries: libchibi-scheme.o

### ISSUE THREAD

**Issue by JacksonGariety** [ashinn/chibi-scheme#379]
When I run `sudo make install` on arch linux:

```
echo "Generating images"
Generating images
cd / && /usr/local/bin/chibi-scheme -d /usr/local/share/chibi/chibi.img
/usr/local/bin/chibi-scheme: error while loading shared libraries: libchibi-scheme.so.0: cannot open shared object file: No such file or directory
make: [Makefile:352: install] Error 127 (ignored)
cd / && /usr/local/bin/chibi-scheme -mchibi.snow.commands -mchibi.snow.interface -mchibi.snow.package -mchibi.snow.utils -d /usr/local/share/chibi/snow.img
/usr/local/bin/chibi-scheme: error while loading shared libraries: libchibi-scheme.so.0: cannot open shared object file: No such file or directory
make: [Makefile:353: install] Error 127 (ignored)
```

Any ideas?

UPDATE:

Realized that tests weren't passing on master so I checked out 0.7.3 (commit 5ddef3a) and `sudo make install` succeeded, however, when I run `chibi-scheme` I get:

```
/usr/local/bin/chibi-scheme: error while loading shared libraries: libchibi-scheme.so.0: cannot open shared object file: No such file or directory
```

**Comment by ashinn** [comment]
You're installing into /usr/local, but you don't have this in your lib path.

I've fixed this in efcb12d8e30f14c695d27fafe71f44157f86b629, but you'll still need to set the LD_LIBRARY_PATH=/usr/local/lib if you want to run chibi-scheme yourself, which is the error when you back up to the previous commit.

**Comment by JacksonGariety** [comment]
Thanks!
